java/arcs/core/storage/referencemode/BridgingOperation.kt

```diff
@@ -82,6 +83,16 @@ sealed class BridgingOperation : CrdtOperationAtTime {
     ) : BridgingOperation() {
         override val clock: VersionMap = containerOp.clock
     }
+
+    /** Denotes a clear of the [CrdtSet] managed by the store. */
+    class ClearSet /* internal */ constructor(
+        override val containerOp: CrdtSet.Operation.Clear<Reference>,
+        override val refModeOp: RefModeStoreOp.SetClear
+    ) : BridgingOperation() {
+        override val entityValue: RawEntity? = null
```

Comment:
one more ask, when applying this it would be good to do the same as RemoveFromSet and call clearEntityInBackingStore() for all entities in the container
http://google3/third_party/java_src/arcs/java/arcs/core/storage/ReferenceModeStore.kt?l=265&rcl=319285219

it is important because fedex uses this to remove all entities, we must make sure all pii is gone from the db after this op.

It can be done in a follow up pr but we wouldn't want to cut a release between this pr and that one. If you want to do it in a follow up please revert changes to collectionhandle in this pr, and do those only after we address this.
let me know if anything is not clear

Reply:
Done. I've also updated a bunch of unit tests.